src/app/login/page.tsx

```diff
@@ -1,12 +1,24 @@
 import LoginButton from "@/components/login-button";
+import SampleActionButton from "@/components/sample-action-button";
```

Comment:
naming conventions in React is unopinionated but maybe we should use widely used naming conventions for files and such. For components, it would be good if they are on PascalCase. this is just my two cents
https://www.linkedin.com/pulse/react-js-naming-convention-kristiyan-velkov/

Reply:
Hey there~ Next.js/Vercel uses kebab-case as convention. 

Sample: https://github.com/vercel/ai-chatbot/tree/main/components

In Next.js API for example...
![image](https://github.com/user-attachments/assets/af478ce2-d002-4495-9926-5a4fb157f424)

shadcn also uses kebab-case.

Other benefits:

1. Eliminates potential issues with case-insensitive file systems/OS
2. Makes "module"/"namespace" style naming easier to read ([example](https://www.reddit.com/r/reactjs/comments/s2wbfx/comment/icioztf/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button))